pkg/build/builder/util/consts.go

```diff
@@ -0,0 +1,40 @@
+package util
+
+const (
+
+	// OriginVersion is an environment variable key that indicates the version of origin that
+	// created this build definition.
+	OriginVersion = "ORIGIN_VERSION"
```

Comment:
Are these guaranteed in any way for custom builds?

Reply:
sort of?  we document it here:
https://docs.openshift.org/latest/creating_images/custom.html#custom-builder-image

Reply:
> sort of? we document it here:
> https://docs.openshift.org/latest/creating_images/custom.html#custom-builder-image

That makes it sound a lot like an API constant.  Doesn't need to move here, but if it is guaranteed, we can't change it, and we expect clients to consume it, then it is a good target for moving to openshift/api.